uniprotkb-rest/src/main/resources/application.properties

```diff
@@ -76,19 +80,19 @@ voldemort.uniprot.storeName=uniprotkb
 streamer.uniprot.storeBatchSize=5000
 streamer.uniprot.storeFetchMaxRetries=5
 streamer.uniprot.storeFetchRetryDelayMillis=500
-#streamer.uniprot.zkHost=wp-np2-b9.ebi.ac.uk:2191,ves-hx-ee.ebi.ac.uk:2191,ves-hx-c9.ebi.ac.uk:2191
-#streamer.uniprot.zkHost=wp-np2-b3:5191,wp-np2-b4:5191,wp-np2-b5:5191
-streamer.uniprot.zkHost=wp-np3-db:5191
```

Comment:
I think we could keep this line, this way we can easily switch back to the new solr env